### DISCUSSION TITLE
DELPH-IN 2025 Summit poll -- please reply by November 15

### DISCUSSION THREAD

**Discussion by emilymbender** [delph-in/docs#154] [Announcements]
Dear all (@delph-in/participants),

It's time to schedule our Summit for next summer! We are fortunate to have two offers to host, from Rio and Amsterdam. Please visit the link below and indicate which week-location pairs would be feasible for you:

https://docs.google.com/spreadsheets/d/1Y9sKbOLfLuZMJqxWNMp_Fy9oSEAjxe25ctFgZEKoA60/edit?usp=sharing

Please reply by **November 15.**

-- Emily, for the Standing Committee

**Comment by emilymbender** [comment]
Reminder to please fill out this poll!

https://docs.google.com/spreadsheets/d/1Y9sKbOLfLuZMJqxWNMp_Fy9oSEAjxe25ctFgZEKoA60/edit?usp=sharing

Please reply by November 15.

-- Emily, for the Standing Committee

**Comment by fcbond** [comment]
Dear all (@delph-in/participants),

last call!  Help us decide when and where to schedule DELPH-IN, ...

https://docs.google.com/spreadsheets/d/1Y9sKbOLfLuZMJqxWNMp_Fy9oSEAjxe25ctFgZEKoA60/edit?usp=sharing

If you haven't already, please reply by the end of November.

-- Francis, for the Standing Committee
